### PR TITLE
Add video to coreos post.

### PR DESCRIPTION
**Pull request by rwjblue**
The video is now available, so I linked it up (adding headers for the video and the slide).

### CODE REVIEW

File: coreos_2015.md [hunk 1]
```diff
@@ -9,8 +9,15 @@ section: Blog
 posts: true
 ---
 
-<script async class="speakerdeck-embed" data-id="7fa1f7d4a794480093dd476ac805bfdd" data-ratio="1.77777777777778" src="//speakerdeck.com/assets/embed.js"></script>
 
 Frank spoke at CoreOS Fest today, about how Aptible uses containers to address security and compliance objectives. The video should be up shortly. In the meantime, visit [the CoreOS Fest website](https://coreos.com/fest/) to see some of the other topics and speakers.
```
Comment (chasballew) [inline coreos_2015.md]: Can you change this to:

```
Frank spoke at CoreOS Fest today, about how Aptible uses containers to address security and compliance objectives. [UPDATE: Check out the video below, and visit [the CoreOS Fest website](https://coreos.com/fest/) to see some of the other topics and speakers.]
```

**Comment by chasballew** [comment]
Awesome, thank you!

File: coreos_2015.md (hunk 1 above)
Comment (rwjblue) [inline coreos_2015.md]: Updated, but slightly tweaked to make the `UPDATED:` bolded.  Let me know if you want me to revert that part...

**Comment by rwjblue** [comment]
screenshot

**Pull request merged**